Unix/share/omischema/CIM-2.32.0/Interop/CIM_CIMOMStatisticalData.mof

```diff
@@ -16,7 +16,7 @@
        "the difference of two snapshots of a counter at the beginning "
        "and end of a measurement interval should get the correct "
        "result, even if there was a wrap-around in between obtaining "
-       "the two snapshots. (Two or more wrap arounds will result in "
+       "the two snapshots. (Two or more wraparounds will result in "

```

Comment:
`wraparound` is a word, as is its plural, but `arounds` isn't generally used.

Fwiw, I'm skipping past `CIM-2.32.0`.